documentation/book/con-kafka-connect-authentication.adoc

```diff
@@ -48,7 +48,7 @@ To use the authentication using the SCRAM-SHA-512 SASL mechanism, set the `type`
 SCRAM-SHA-512 uses a username and password to authenticate.
 Specify the username in the `username` property.
 Specify the password as a link to a `Secret` containing the password in the `passwordSecret` property.
-It has to specify the name of the `Secret` containing the password and the name of the key under which the password is stored inside the `Secret`.
+It has to specify the name of the `Secret` in the `secretName` property and the name of the key under which the password is stored inside the `Secret` in the `password` property.
```

Comment:
Should we add some explicit warnings here and to the Mirror Maker part? Such as

```
WARNING: The `password` field should not contain the actual password. It should contain the key under which it is stored in the `Secret`!
```

Reply:
Well the "name of the key under which the password is stored inside the `Secret` in the `password` property." part should be clear about that but we know people, so I am fine to add this warning.

Reply:
@ppatierno - Please can we reword the warning. No exclamation marks, please!

WARNING: Do not specify the password in the `password` field. 

Are you sure we need to elaborate further? Can we rename the `password` field to something more descriptive?

Reply:
I'll remove question mark rephrasing the warning but we cannot change the `password` field because it's related to the need for a software change.